apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/ReadTests.cs

```diff
@@ -63,28 +63,17 @@ private async Task<T> ExecuteAsync<T>(string sql)
 
         [Fact]
         public async Task BadColumnName()
-        {
-            int rowsRead = -1;
-            var e = await Assert.ThrowsAsync<SpannerException>(
-                async () =>
                 {
                     using (var connection = _fixture.GetConnection())
                     {
                         var cmd = connection.CreateSelectCommand($"SELECT badjuju FROM {_fixture.TableName}");
                         using (var reader = await cmd.ExecuteReaderAsync())
                         {
-                            rowsRead = 0;
-                            while (await reader.ReadAsync())
-                            {
-                                rowsRead++;
-                            }
-                        }
-                    }
-                }).ConfigureAwait(false);
-
+                    var e = await Assert.ThrowsAsync<SpannerException>(() => reader.ReadAsync());
```

Comment:
nit: Indentation in this whole block is off, or at least it looks like it on github.

Reply:
Wow, yes. That's really bizarre. I think this is due to committing just selected lines. Will check all the files in that commit, and rebase it after fixing.

Reply:
Okay, I *think* that's now okay... will wait for it to go green in Travis though.